packages/nextjs-mf/src/loaders/nextPageMapLoader.ts

```diff
@@ -54,6 +63,15 @@ export function exposeNextjsPages(cwd: string) {
     };
 }
 
+/**
+ * This function gets the root directory of the NextJS pages.
+ * It checks if the 'src/pages/' directory exists.
+ * If it does, it returns the absolute path and the relative path to this directory.
```

Comment:
The function 'preparePageMapV2' can be improved by using a map function instead of a for loop. This can improve readability and maintainability.